packages/epo-react-lib/src/atomic/Buttonish/Buttonish.tsx

```diff
@@ -3,14 +3,16 @@ import Link from "next/link";
 import { isInternalUrl } from "@/helpers/index";
 import Button, { ButtonProps } from "@/atomic/Button";
 
-interface ButtonishProps extends Omit<ButtonProps, "children"> {
+interface ButtonishProps extends ButtonProps {
```

Comment:
doing the above also means we can simply extend the `ButtonProps` without omitting children.

Reply:
Very neat.